projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -31,32 +35,74 @@ public ProjectCreationService(ProjectRepository projectRepository) {
    */
   public Result<Project, ApplicationException> createProject(String title, String objective) {
     ProjectTitle projectTitle;
+    try {
+      projectTitle = ProjectTitle.create(title);
+    } catch (RuntimeException e) {
+      log.error(e.getMessage(), e);
+      return Result.failure(new ProjectManagementException(ErrorCode.INVALID_PROJECT_TITLE,
+          ErrorParameters.of(200, title)));
```

Comment:
Not sure I got the meaning of the code `200`

Reply:
This is not a code but the max length of project titles (stubbed, needs to come from the title once implemented)